Header/indexHeader.css

```diff
@@ -0,0 +1,111 @@
+header {
+    /* background-image: linear-gradient(rgba(0,0,0,0.6),rgba(0,0,0,0.7)), url('https://harttohart.com/wp-content/uploads/2017/12/black-background-1468370534d5s.jpg'); */
+    height: 100vh;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    background-color: var(--bg-primary);
+}
+
+.clearfix :after {
+    content: ".";
+    visibility: hidden;
+    display: block;
+    height: 0px;
+    clear: both;
+}
+
+.main-content-header>h1 {
+    color: var(--bg-white);
+    font-family: 'Open Sans';
+}
+
+.row {
+    max-width: 1180px;
+    margin: 0 auto;
+}
+
+.logo {
+    width: auto;
+    float: left;
+    margin-top: 60px;
+    
+    
+}
+
+.main-nav {
+    float: right;
+    margin-top: 60px;
+}
+
+.main-nav li {
+    display: inline-block;
+    list-style: none;
+    margin-left: 40px;
+
+}
+
+.main-nav li a {
+    padding: 5px 0;
+    color: var(--bg-white);
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 90%;
+    font-weight: 700
+}
+
+.main-nav li a:hover {
+    border-bottom: 2px solid var(--bg-pink);
+}
+
+.main-content-header {
```

Comment:
You can use `max-width:80%` along with `margin:0 auto` to get a centered container. As I said, avoid `px` as much as you can.